draft-ietf-tls-tls13.md

```diff
@@ -3787,7 +3854,9 @@ is listed below:
    1.3" column with the following four values: "Client", indicating
    that the server shall not send them. "Clear", indicating
    that they shall be in the ServerHello. "Encrypted", indicating that
-   they shall be in the EncryptedExtensions block, and "No" indicating
+   they shall be in the EncryptedExtensions block, "Early", indicating
+   that they shall be in the client's 0-RTT EncryptedExtensions block,
+   and "No" indicating
```

Comment:
Unless you think that we need to allow Early and Client, or Early and Clear, or Early and Encrypted.